src/utility/storage/storage.service.ts

```diff
@@ -10,25 +10,34 @@ import {
   ListObjectsV2Command,
   DeleteObjectCommandInput,
   DeleteObjectCommand,
+  CreateMultipartUploadCommand,
+  CreateMultipartUploadCommandInput,
+  UploadPartCommand,
+  CompleteMultipartUploadCommand,
+  CompleteMultipartUploadCommandInput,
+  UploadPartCommandInput,
 } from '@aws-sdk/client-s3';
 import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
-import { Injectable } from '@nestjs/common'
+import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 
 @Injectable()
 export class StorageService {
   private readonly awsS3RegionStorage: string;
+  private readonly awsS3RegionStatic: string;
   private readonly awsS3BucketStatic: string;
   private readonly awsS3BucketStorage: string;
 
   constructor(
     private readonly configService: ConfigService<{
       AWS_S3_BUCKET_STATIC: string;
+      AWS_S3_REGION_STATIC: string;
```

Comment:
This key is not add into `.env` & `.env.example`  .

Reply:
fixed